rules/docker-dhcp-relay.mk

```diff
@@ -15,17 +15,43 @@ $(DOCKER_DHCP_RELAY)_DBG_IMAGE_PACKAGES = $($(DOCKER_CONFIG_ENGINE_BUSTER)_DBG_I
 
 $(DOCKER_DHCP_RELAY)_LOAD_DOCKERS = $(DOCKER_CONFIG_ENGINE_BUSTER)
 
+$(DOCKER_DHCP_RELAY)_INSTALL_PYTHON_WHEELS = $(SONIC_UTILITIES_PY3)
+$(DOCKER_DHCP_RELAY)_INSTALL_DEBS = $(PYTHON3_SWSSCOMMON)
+
 $(DOCKER_DHCP_RELAY)_VERSION = 1.0.0
 $(DOCKER_DHCP_RELAY)_PACKAGE_NAME = dhcp-relay
+$(DOCKER_DHCP_RELAY)_PACKAGE_DEPENDS = database^1.0.0
 
-SONIC_DOCKER_IMAGES += $(DOCKER_DHCP_RELAY)
```

Comment:
What is the impact of this change ?
Remove from 
SONIC_DOCKER_IMAGES += $(DOCKER_DHCP_RELAY)
SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_DHCP_RELAY)

Reply:
Can you please explain ?
Per prev comments "_It is installed by default and enabled, I don't change that_."
Here it is "_Do not add this docker to a list of regular installed dockers_"

There is a discrepancy. 
Feel free to call me in teams, if that would help

Reply:
@renukamanavalan Sorry, let me clarify this. The key word here is *regular*. It is no longer added to SONIC_INSTALL_DOCKER_IMAGES (as any other docker) but added to SONIC_PACKAGES_LOCAL (new target group handled differently in sonic_debian_extension, basically it will invoke "sonic-package-manager install" for dhcp-relay.)

Reply:
Thanks Stepan!